app/code/Magento/ConfigurableProduct/Plugin/Catalog/Model/Product/Attribute/Backend/TierPrice/UpdateHandlerPlugin.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\ConfigurableProduct\Plugin\Catalog\Model\Product\Attribute\Backend\TierPrice;
+
+use Magento\Catalog\Api\ProductAttributeRepositoryInterface;
+use Magento\Catalog\Model\Product\Attribute\Backend\TierPrice\UpdateHandler;
+use Magento\ConfigurableProduct\Model\Product\Type\Configurable;
+
+/**
+ * Plugin for handling tier prices during product attribute backend update.
+ */
+class UpdateHandlerPlugin
+{
+    /**
+     * @var ProductAttributeRepositoryInterface
+     */
+    public readonly ProductAttributeRepositoryInterface $attributeRepository;
```

Comment:
Please make this a private
```suggestion
    private readonly ProductAttributeRepositoryInterface $attributeRepository;
```

Reply:
Hello @engcom-Hotel ,
I have made changes as per your suggestions.